mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensorAttrDefs.td

```diff
@@ -240,8 +240,9 @@ def SparseTensorEncodingAttr : SparseTensor_Attr<"SparseTensorEncoding",
     // CSR slice (offset = 0, size = 4, stride = 1 on the first dimension;
     // offset = 0, size = 8, and a dynamic stride on the second dimension).
     #CSR_SLICE = #sparse_tensor.encoding<{
-      lvlTypes = [ "dense", "compressed" ],
-      dimSlices = [ (0, 4, 1), (0, 8, ?) ]
+      map = (d0 : #sparse_tensor<slice(0, 4, 1)>,
```

Comment:
can we use "i" and "j" in the documentation? it of course does not matter and we will print it back with d0/d1 most likey (as all the affine stuff), but I want to make the doc very user friendly ;-)

Reply:
Done!